circle.yml

```diff
@@ -44,3 +44,26 @@ deployment:
         --restart=always
         -P bespoken/bst:$CIRCLE_TAG
       - ./hyper fip attach -f 199.245.58.242 bst
+
+  bst-dev-server:
+    tag: /bst-dev-.*/
+    commands:
+      - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
+      - docker build -f docker/Dockerfile -t bespoken/bst-dev:$CIRCLE_TAG .
```

Comment:
I would keep the image name the same (bespoken/bst) - and use the tag to differentiate it (bst-dev-x or bst-server-dev-x). If you look in DockerHub, that is a cleaner way to organize images.